src/screens/Cart/index.tsx

```diff
@@ -173,6 +174,17 @@ export const Cart = () => {
 
       setLoading(false);
       goSuccessStep();
+
+      await queryClient.invalidateQueries({
+        predicate: (query: any) =>
```

Comment:
`any` because I believe typing is broken here and TS says that `query` in `unknown`.